js/scripts.js

```diff
@@ -0,0 +1,135 @@
+$(function() {
+
+// -------------------------------------------------------------------------------------------------
+
+    var board = {
+        name: 'Kanban Board',
+        addColumn: function(column) {
+            this.$element.append(column.$element);
+            initSortable();
+        },
+        $element: $('#board .column-container')
+    };
+
+    $('.create-column')
```

Comment:
jeśli szukasz w JS elementów HTML to najlepiej jest używać ID - jest to najszybszy sposób znajdowania elementów, tym bardziej że w tym wypadku masz tylko jeden button .create-column